### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by paulstapor**
I hope, everything is okay for the documentation. Please look this through closely. When this request is merged, the develop branch can be deleted...

### CODE REVIEW

File: doc/documentation.md [hunk 1]
```diff
@@ -81,24 +81,24 @@ they can be used for parameter estimation.
...
 
-See getParameterSamples() for details and mainConversionReaction.m for examples.
+Details on the recommended settings can be found in PestoSamplingOptions.m and the correspoind option files for each algorithm. Examples for the algorithms can be found in mainConversionReaction.m (PT as single-chain algorithm), mainEnzymaticCatalysis.m (PT as multi-chain algorithm), mainTransfection.m (PHS as multi-chain algorithm) and mainExampleRing.m and mainExampleGauss.m (all algorithms, respectively).
```
Comment (dweindl) [inline doc/documentation.md]: correspoind

File: examples/examples_documentation.md [hunk 2]
```diff
@@ -8,15 +8,16 @@ Examples
...
-* *mRNA transfection* § (`examples/mRNA_transfection/mainTransfection.m`, ODE)
-* *Pom1p gradient formation* § (`examples/Pom1p_gradient_formation/mainPom1.m`, PDE)
-* *Jak-Stat-signaling* § (`examples/jakstat_signaling/mainJakstatSignaling.m`, ODE)
+* *Conversion reaction*, the simplest example and demonstrating most of the PESTO features (`examples/conversion_reaction/mainConversionReaction.m`, ODE)
+* *mRNA transfection*, another very simple example, also demonstrating many of the PESTO features (`examples/mRNA_transfection/mainTransfection.m`, ODE)
+* *Enzymatic catalysis*, the third small example case, also demonstrating a variety of the PESTO features (`examples/enzymatic_catalysis/mainEnzymaticCatalysis.m`, ODE)
+* *Gaussian mixture*, a benchmark model for the MCMC sampling routines in PESTO (`examples/GaussExample/mainExampleGauss.m`)
+* *Hyperring*, a second benchmark model (with non-identifiabilities) for the MCMC sampling routines in PESTO, (`examples/RingExample/mainExampleRing.m`)
+* *Pom1p gradient formation*, a PDE problem which is discertized in space and reformulated as ODE § (`examples/Pom1p_gradient_formation/mainPom1.m`, PDE)
```
Comment (dweindl) [inline examples/examples_documentation.md]: discretized

File: examples/examples_documentation.md [hunk 3]
```diff
@@ -27,18 +28,30 @@ The following table provides an overview of which of the PESTO functions are dem
...
+| getPropertySamples()      |          X          |                     |                  |     X     |      X       |       |         |      |
+| plotPropertySamples()     |          X          |                     |                  |     X     |      X       |       |         |      |
+| getPropertyConfidenceIntervals() |   X          |                     |                  |           |      X       |       |         |      |
 | testGradient()            |                     |                     |                  |           |              |       |         |      |
 | collectResults()          |                     |                     |                  |           |              |       |         |      |
+
+In addition to the principal routines, the examples also highlight how analysis results may look like in certain situations (especially in the case of modelling related problems), or how the results from the PESTO routines can be manipulated (the following list is of course non-exhaustive):
+* *multi-modal posterior distributions* (transfection and Gaussian mixture)
+* *structurally non-identifiable problems* (Hperring and transfection)
```
Comment (dweindl) [inline examples/examples_documentation.md]: Hyperring

File: examples/examples_documentation.md [hunk 4]
```diff
@@ -27,18 +28,30 @@ The following table provides an overview of which of the PESTO functions are dem
...
+* *practically non-identifiable problems* (enzymatic catalysis)
```
Comment (dweindl) [inline examples/examples_documentation.md]: non-identifiable problem or rather non-identifiable parameters?

Comment (paulstapor) [inline examples/examples_documentation.md]: model with non-identifiabilities? I was also wondering what sounds best...

Comment (dweindl) [inline examples/examples_documentation.md]: "non-identifiabilities" or "non-identifiable parameters", no preference

**Pull request merged**